helper/resource/testing_new_import_state.go

```diff
@@ -105,34 +105,36 @@ func testStepNewImportState(ctx context.Context, t testing.T, helper *plugintest
 		}
 	}
 
+	var inlineConfig string
+	if step.Config != "" {
+		inlineConfig = step.Config
+	} else {
+		inlineConfig = cfgRaw
+	}
 	testStepConfigRequest := config.TestStepConfigRequest{
 		StepNumber: stepNumber,
 		TestName:   t.Name(),
 	}
 	testStepConfig := teststep.Configuration(teststep.PrepareConfigurationRequest{
 		Directory:             step.ConfigDirectory,
 		File:                  step.ConfigFile,
-		Raw:                   step.Config,
+		Raw:                   inlineConfig,
 		TestStepConfigRequest: testStepConfigRequest,
 	}.Exec())
 
-	if testStepConfig == nil {
-		logging.HelperResourceTrace(ctx, "Using prior TestStep Config for import")
-		importConfig := cfgRaw
+	switch {
+	case step.ConfigExact:
+		break
```

Comment:
Does this fallthrough by default if there is no body? My yearly reminder of the `fallthrough` Go directive 😆 

I like the explicitness regardless

Reply:
Nice callout 😃 `break` is indeed optional here. I likely wrote `break` explicitly due to habit. To your point, yes, explicitness does feel more intention-revealing here & kinder to our future selves.

I've confirmed that the tests pass without the `break` statement, and they fail when I replace it with `fallthrough`.